src/kratos.ts

```diff
@@ -60,78 +59,75 @@ export const clearAuthentication = async (
   req: FastifyRequest,
   res: FastifyReply,
   reason: string,
-): Promise<void> =>
-  runInSpan('clearAuthentication', async () => {
-    req.log.info(
-      {
-        reason,
-        userId: req.userId,
-        cookie: req.cookies[cookies.kratos.key],
-      },
-      'clearing authentication',
-    );
-    req.trackingId = await generateTrackingId();
-    req.userId = undefined;
-    setTrackingId(req, res, req.trackingId);
-    setCookie(req, res, 'auth', undefined);
-    setCookie(req, res, 'kratosContinuity', undefined);
-    setCookie(req, res, 'kratos', undefined);
-  });
+): Promise<void> => {
+  req.log.info(
```

Comment:
Can you add a metric please with the reason?

Reply:
Yeah, I will add a metric to this

Reply:
This is the only thing left right?

Reply:
Yeah. Currently working on it.

Reply:
Added metric counter with the reason now